swarm_rl/env_wrappers/compatibility.py

```diff
@@ -43,7 +43,7 @@ def step(self, action: Any) -> Tuple[Any, float, bool, bool, Dict]:
         """
         obs, reward, done, info = self.env.step(action)
 
-        return convert_to_terminated_truncated_step_api((obs, reward, done, info), is_vector_env=True)
+        return obs, reward, done, False, info #convert_to_terminated_truncated_step_api((obs, reward, done, info), is_vector_env=True)
```

Comment:
We should not comment out this line, since this wrapper is for the updated version of gym. You mentioned that the reason for commenting out is to get the tests to work in quadrotor_single. Please think about another way to fix that.